packages/server/src/modules/event/EventType.js

```diff
@@ -1,31 +1,31 @@
 import { globalIdField } from "graphql-relay";
 import { connectionDefinitions } from "../CustomConnectionType";
-import { nodeInterface } from "../../type/QueryType";
+import { nodeInterface, registerType } from "../../type/QueryType";
 import {
   GraphQLObjectType,
   GraphQLString,
   GraphQLBoolean,
   GraphQLNonNull
 } from "graphql";
 
-console.log(connectionDefinitions);
-const EventType = new GraphQLObjectType({
-  name: "Event",
-  fields: () => ({
-    id: globalIdField("events"),
-    _id: { type: GraphQLString },
-    title: { type: GraphQLString },
-    date: { type: GraphQLString },
-    description: { type: GraphQLString },
-    author: {
-      type: GraphQLString,
-      resolve: (obj, args, context, info) => {
-        return obj.author.name;
+const EventType = 
+  new GraphQLObjectType({
+    name: "Event", // It seems some error with this type, which "name" is not found, but "default" is
+    fields: () => ({
+      id: globalIdField("events"),
+      _id: { type: GraphQLString },
+      title: { type: GraphQLString },
+      date: { type: GraphQLString },
+      description: { type: GraphQLString },
+      author: {
+        type: GraphQLString,
+        resolve: (obj, args, context, info) => {
+          return obj.author.name;
```

Comment:
did this work?

Reply:
author should be an _user id_
and you should do a `UserLoader.load`
and the return type should be an `UserType`

Reply:
this is a temporary workaround
now that there is DataLoaders it should be easier to achieve this behavior